ard/farm_aero/floris.py

```diff
@@ -238,13 +238,32 @@ def get_thrust_turbines(self):
         """Get the turbine thrusts of a FLORIS farm at each wind condition."""
         # FLORIS computes the thrust precursors, compute and return thrust
         # use pure FLORIS to get these values for consistency
+
+        # prepare to unpack thrust data that is not pre-computed in FLORIS
         CT_turbines = self.fmodel.get_turbine_thrust_coefficients()
         V_turbines = self.fmodel.turbine_average_velocities
         rho_floris = self.fmodel.core.flow_field.air_density
         A_floris = np.pi * self.fmodel.core.farm.rotor_diameters**2 / 4
 
+        # unpacking procedure
+        # from FLORIS's floris_model.py:564 at a6fc5d35aa32614edc450dc399c42af60a816887
         thrust_turbines = CT_turbines * (0.5 * rho_floris * A_floris * V_turbines**2)
-        return thrust_turbines.T
+        if isinstance(self.fmodel.wind_data, floris.WindRose) or isinstance(
```

Comment:
Interesting this is needed, but otherwise no problem. If there is a common access method that would be better.